idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/gas_phase_thermo.py

```diff
@@ -531,21 +624,21 @@ def _visc_d(self):
                           units=pyunits.kg/pyunits.m/pyunits.s)
 
         def visc_d_comp(i):
-            return self._params.visc_d_param[i, 1] * \
-                    (self.temperature**self._params.visc_d_param[i, 2]) \
-                    / ((1 + (self._params.visc_d_param[i, 3]/self.temperature))
-                        + (self._params.visc_d_param[i, 4] /
-                           (self.temperature**2)))
+            return (self._params.visc_d_param_1[i] *
+                    ((self.temperature/pyunits.K) **  # need /K to cancel units
```

Comment:
Division by pyunits.K will no longer be necessary when the units for visc_d_param_1 are updated per prior comment.